lab_1/main.py

```diff
@@ -25,7 +27,13 @@ def remove_stop_words(tokens: list, stop_words: list) -> list:
     stop_words = ['the', 'is']
     --> ['weather', 'sunny', 'man', 'happy']
     """
-    pass
+    tokens = ['the', 'weather', 'is', 'sunny', 'the', 'man', 'is', 'happy']
```

Comment:
You should not define the `tokens` here - they are coming as an argument. 
You should filter out the words that are in `stop_words` from the `tokens`.